### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

**Comment by chevtche** [comment]
retest this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,11 +12,15 @@
 
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 project(Livre VERSION 0.8.0)
-set(Livre_VERSION_ABI 8)
+set(Livre_VERSION_ABI 7)
```
Comment (hernando) [inline CMakeLists.txt]: Why you have decreased the ABI number?

Comment (chevtche) [inline CMakeLists.txt]: Because  it was increased two time

**Comment by chevtche** [comment]
There was no build instructions in the readme but I can add them.

**Comment by chevtche** [comment]
retest this please

File: CMakeLists.txt (hunk 1 above)
Comment (hernando) [inline CMakeLists.txt]: But then the version number should be decreased, shouldn't it?

**Comment by hernando** [comment]
Retest this please

File: CMakeLists.txt (hunk 1 above)
Comment (hernando) [inline CMakeLists.txt]: OK, I verified myself, last release was already 7 and 6

**Comment by hernando** [comment]
Retest this please

**Comment by chevtche** [comment]
retest this please

**Pull request merged**